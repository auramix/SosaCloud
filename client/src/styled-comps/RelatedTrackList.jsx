import styled from 'styled-components';


const RelatedTrackList = styled.ul`
  padding: 0;
  margin: 0;
  display: block;
  list-style: none;
  text-align: left;
`;

export default RelatedTrackList;