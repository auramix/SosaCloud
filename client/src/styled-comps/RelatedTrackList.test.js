import RelatedTrackList from './RelatedTrackList.jsx';

describe('RelatedTrackList', () => {
  it('Has correct number of children', () => {
    const wrapper = shallow(<RelatedTrackList />);

    expect(wrapper.children()).toHaveLength(0);
  });

  // Snapshot test
  it('Has expected snapshot output', () => {
    const wrapper = shallow(
      <RelatedTrackList />
    );
    expect(wrapper).toMatchSnapshot();
  })

});