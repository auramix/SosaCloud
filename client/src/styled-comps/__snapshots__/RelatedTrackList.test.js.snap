// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RelatedTrackList Has expected snapshot output 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  padding: 0;
  margin: 0;
  display: block;
  list-style: none;
  text-align: left;
",
        ],
      },
      "displayName": "styled.ul",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "ul",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
